src/main.ts

```diff
@@ -68,24 +68,35 @@ class LocalTableland {
     }
 
     // make sure we are starting fresh
-    // TODO: I don't think this is doing anything anymore...
     this.#_cleanup();
 
-    // Run a local hardhat node
-    this.registry = spawn(
-      isWindows() ? "npx.cmd" : "npx",
-      ["hardhat", "node"],
-      {
-        // we can't run in windows if we use detached mode
-        detached: !isWindows(),
-        cwd: this.registryDir,
-        env: {
-          ...process.env,
-          HARDHAT_NETWORK: "hardhat",
-          HARDHAT_UNLIMITED_CONTRACT_SIZE: "true",
-        },
+    // There's two ways of signaling a mainnet fork should be used. The `FORK`
+    // env var, or the config has a fork property.  Either way this means we
+    // don't need to deploy the registry, and the validator should listen to
+    // a different contract address, specifically the mainnet address.
+    const shouldFork = !!(process.env.FORK || config.fork);
+    const hardhatCommandArr = ["hardhat", "node"];
+
+    if (config.fork) {
+      hardhatCommandArr.push("--fork");
+      hardhatCommandArr.push(config.fork);
+      if (config.forkBlockNumber) {
+        hardhatCommandArr.push("--fork-block-number");
```

Comment:
This is where the JS class config is set, but the dev can also use the command flags or env vars to specify.